QUICKSTART.md

```diff
@@ -0,0 +1,36 @@
+# LAMP stack development environment quickstart
```

Comment:
Why limit it to D8?  While many of these changes are for D8 compatibility it works for any modern LAMP stack app.